src/components/HeaderWithLanguage/HeaderActions.tsx

```diff
@@ -118,6 +131,7 @@ const translatableTypes = [
   "image",
   "podcast-series",
   "frontpage-article",
+  "compare",
```

Comment:
Alt anna er datatyper vi kan redigere i ed bortsett fra denne. Tenker det kanskje er bedre å ikkje ha header for compare-sida for ikkje å rote ting til. Akkurat som forhåndsvisningssida.

Reply:
Jonas mente det var ryddig med en h1, og jeg er egentlig enig med han (men om dere begge lander på at vi ikke skal ha det allikevel, kan jeg ta den bort igjen).
Jeg kan alternativt sette den opp hakket mer manuelt heller enn å bruke HeaderWithLanguage-komponenten, dersom vi er redde for at denne skal gå i beina te resten.

Reply:
Vi har jo allerede to h1 på sida, nemlig artikkeloverskriftene. Treng vel ikkje en ekstra h1 på toppen då.